modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/test/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormAdminDisplayContextTest.java

```diff
@@ -129,6 +130,31 @@ public void testGetFormURLForSharedFormInstance() throws Exception {
 		Assert.assertEquals(getSharedFormURL(), formURL);
 	}
 
+	@Test
+	public void testGetPublishedURLForPublishedForm() throws Exception {
+		DDMFormInstance publishedDDMFormInstance = mockDDMFormInstance(
+			_SHARED_FORM_INSTANCE_ID, true, false);
```

Comment:
@marcosapmf, can we add the boolean values to local variables as we did in the JavaScript tests?